Code/Framework/AzToolsFramework/AzToolsFramework/API/EntityPropertyEditorRequestsBus.h

```diff
@@ -31,6 +33,10 @@ namespace AzToolsFramework
         //! Explicitly sets a component as having been the most recently added.
         //! This means that the next time the UI refreshes, that component will be ensured to be visible.
         virtual void SetNewComponentId(AZ::ComponentId componentId) = 0;
+
+        //! Provides access to the component editors in an EntityPropertyEditor via a callback.
+        //! @param callback The callback that iterates over all the component editors within an EntityPropertyEditor.
+        virtual void GetComponentEditors(const GetComponentEditorsCallback& callback) const = 0;
```

Comment:
suggestion: Should this potentially be called something like `Process` or `Update` rather than `Get` (as no value is being returned and we're iterating and potentially modifying the ComponentEditors?)

Reply:
Yes, I like visit better than 'Process' or 'Update' because the callback merely provides access to the component editors by iterating over the vector, it doesn't modify or update them. I'll change it to VisitComponentEditors() since that'll be more aligned to the visitor pattern being implemented here.

Reply:
Nice good call! 👍